### DISCUSSION TITLE
tracking remix release changes has become hell

### DISCUSSION THREAD

**Discussion by akomm** [remix-run/remix#3750] [General]
I'm currently on 1.3.3 and wanted to check what changed. It was previously already spammed fully with this:
![Screenshot 2022-07-15 at 10-28-03 Discussions · remix-run_remix](https://user-images.githubusercontent.com/2409779/179185409-d1b266d0-d134-467d-948a-ce052414678b.png)

But now instead of (because its monorepo and you up the versions synchronously) writing at least the changes in the release, you also delegate to every single package individually:

![Screenshot 2022-07-15 at 10-33-27 remix-run_remix Build Better Websites  Create modern resilient user experiences with web fundamentals](https://user-images.githubusercontent.com/2409779/179185765-1897bc2f-00ef-43fa-a246-5bf485904c81.png)

I wonder, did I miss something, or is this how its going to be in every release?
I used search on the docs page in hope that there is an overview of some sort, but didn't find any.

Thanks in advance

**Comment by akomm** [comment]
One thing I'v just noted. I usually switched from releases to tags view, becaue the experimentals also appeared in released. But it appears the release view is now clear of the experimentals. I'm pretty sure it was not before, the reason why I switched to the more compact tags view. Thats good.

**Comment by sergiodxa** [comment]
I agree, I watch the repo for releases to know when there's a new release and read the change log, so I know what changed before upgrading, now it's really hard and every release creates a lot of notifications.

I wonder if the docs could track the change log in an automated way and show it there maybe, so instead of keeping track of the GitHub release it should be possible to read on the docs the stable ones.

**Reply by kiliman** [reply]
I understand the desire to automate as much as possible, but I think release notes is something that would benefit from the human touch. I'm sure there are many people that would volunteer to consolidate the CHANGELOGs into a more useful document.

**Reply by moishinetzer** [reply]
> I understand the desire to automate as much as possible, but I think release notes is something that would benefit from the human touch. I'm sure there are many people that would volunteer to consolidate the CHANGELOGs into a more useful document.

👋 if devs see this please reach out to me, i'm more than happy to do this

**Reply by sergiodxa** [reply]
> I understand the desire to automate as much as possible, but I think release notes is something that would benefit from the human touch. I'm sure there are many people that would volunteer to consolidate the CHANGELOGs into a more useful document.

I said to do it in an automated way to don't add more work to the team, and since the release notes are already written in the GitHub release in theory if they filter non stable releases they should be able to get all release notes.

But if it needs to be done by hand and it's not a lot more of work that would work yeah.

**Comment by akomm** [comment]
Looks like someone has added changes to the release (or at least improved it with link to packages that have changes too). Thanks!

https://github.com/remix-run/remix/releases/tag/remix%401.6.5
